templates/ai/index.html

```diff
@@ -0,0 +1,688 @@
+{% block content %}
+
+<section class="p-strip--suru-topped is-dark is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>AI on Ubuntu: the path<br>to production</h1>
+      <h4>The best data science and MLops tools across your infra</h4>
+      <p>Develop AI models on high-end Ubuntu workstations. Train on racks of bare-metal or public clouds with hardware acceleration. Deploy to cloud, edge and IoT. All on Ubuntu.</p>
+      <p>
+        <a href="https://charmed-kubeflow.io/docs/install" class="p-button--positive p-link--external">Try Kubeflow</a>
+        <a href="/ai/consulting" class="p-button--neutral">AI services</a>
+      </p>
+    </div>
+    <div class="col-6 u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/cc376f8c-AI-models.svg",
+        alt="",
+        width="304",
+        height="263",
+        hi_def=True,
+        loading="auto|lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <p class="p-muted-heading u-align--center">LEADERS IN ARTIFICIAL INTELLIGENCE CHOOSE UBUNTU</p>
+  <div class="row">
+    <ul class="p-inline-images--small u-no-margin--bottom">
+      <li class="p-inline-images__item">
+        {{ image(
+          url="https://assets.ubuntu.com/v1/ae0456eb-Netflix_logo.svg",
+          alt="Netflix",
+          width="130",
+          height="36",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/1095d3bc-Tesla_logo_silver.png",
+          alt="Tesla",
+          width="130",
+          height="17",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+          }}
+      </li>
+      <li class="p-inline-images__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/52d4cc16-Nvidia_Logo_Horizontal.svg",
+          alt="Nvidia",
+          width="130",
+          height="25",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/26eb00dd-fb.png",
+          alt="Facebook",
+          width="130",
+          height="25",
+          hi_def=True,
+          loading="auto|lazy"
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/c346c103-openai.png",
+            alt="OpenAI",
+            width="130",
+            height="90",
+            hi_def=True,
+            loading="auto|lazy"
+            ) | safe
+          }}
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Develop</h1>
```

Comment:
I think these headings should be h2 going from the design, (there should only be one h1 on the page)
```suggestion
      <h2>Develop</h2>
```

Reply:
Fixed!